roles/splunk/tasks/configure_auditd.yml

```diff
@@ -0,0 +1,45 @@
+---
+- name: "get {{ splunk_nix_user }} user uid"
```

Comment:
Will this work if `splunk_nix_user` is not a local account?

Reply:
I never even thought about that. Is that even something we have to worry about. The role has always created local accounts and we don't use ldap atm for a central account repository. 

I'll reach out to some of our other teams I know that have non-local accounts and see what I can find out.